tests/unit/test.program.js

```diff
@@ -32,7 +36,9 @@ describe('program.Program', () => {
     let thing = spy(() => Promise.resolve());
     let program = new Program(['thing'])
       .command('thing', 'does a thing', thing);
-    return execProgram(program)
+    return execProgram(program, {
+      checkForUpdates: spy(),
```

Comment:
It wasn't being passed by default in `execProgram()` and making some of the tests fail. Not sure why.